pkg/commands/deps.go

```diff
@@ -110,12 +120,34 @@ If the image was not built using ko, or if it was built without embedding depend
 					return err
 				}
 				cmd := exec.CommandContext(ctx, "go", "version", "-m", n)
-				cmd.Stdout = os.Stdout
+				var buf bytes.Buffer
+				cmd.Stdout = &buf
 				cmd.Stderr = os.Stderr
-				return cmd.Run()
+				if err := cmd.Run(); err != nil {
+					return err
+				}
+				// In order to get deterministics SBOMs replace
+				// our randomized file name with the path the
+				// app will get inside of the container.
+				mod := bytes.Replace(buf.Bytes(),
+					[]byte(n),
+					[]byte(path.Join("/ko-app", filepath.Base(filepath.Clean(h.Name)))),
+					1)
```

Comment:
You should be able to remove the `sed -1d` in the actions flow with this 🤩

Reply:
https://github.com/google/ko/blob/6d0691330a6df9407a1232e8e8835c6860387b22/.github/workflows/kind-e2e.yaml#L89-L90